modules/authentication/src/admin/user.ts

```diff
@@ -86,6 +86,9 @@ export class UserAdmin {
         'Can not enable 2fa without a phone number',
       );
     }
+    if (twoFaMethod === 'qrcode') {
```

Comment:
Make this `!== 'phone'` just in case we support additional options down the road.